Dockerfile

```diff
@@ -4,7 +4,7 @@
 # not be updating this dependency.  See
 # https://mesosphere.slack.com/archives/C4E91G0CX/p1541505296001800 for more
 # background.
-FROM mesosphere/dcos-commons-base@sha256:e9f67d72aa1431f4a15544981285f028decc7507538f8e5aa8b0888a94d820c0
+FROM mesosphere/dcos-commons-base:af8469627a8cc4c823fac826a5e02f0acaf07869
```

Comment:
Is this not a step in the wrong direction with regards to managing these dependencies?

Reply:
Does this meant that you built and pushed this image separately?

Reply:
@elezar that was the strategy used here, this PR builds the dcos-commons-base image: https://github.com/mesosphere/dcos-commons/pull/2784

Here we are pointing to the built image, with the other updates.

I've pushed up a commit to use the sha256 hash as opposed to the git-hash which was used before for pulling the correct version of the image.